src/leiningen/test_children.clj

```diff
@@ -0,0 +1,56 @@
+;; Copyright (c) 2016 BigML, Inc
+;; All rights reserved.
```

Comment:
are you thinking of making this available in clojars? if so, you'll want to harmonize licenses...